src/Cimpress.Auth0.Client/Proxies/RoAuthenticationRequestDto.cs

```diff
@@ -8,14 +8,14 @@ namespace Cimpress.Auth0.Client.Proxies
     /// <remarks>
     /// This has been copied and slightly adapterd from https://github.com/auth0/auth0.net since it doesn't yet support netcore.
     /// </remarks>
-    public class AuthenticationRequestDto
+    public class RoAuthenticationRequestDto
```

Comment:
no, no, I like this, what got me confused is that it was named something different before the PR, and I was trying to figure out why we changed it.  I like what you did.